docs/semantics.md

```diff
@@ -95,7 +95,16 @@ The negative result for ```foo/``` will be cached, but that only helps with the
 To alleviate this, Cloud Storage FUSE supports a "type cache" on directory inodes. When type cache is enabled, each directory inode will maintain a mapping from the name of its children to whether those children are known to be files or directories or both. When a child is looked up, if the parent's cache says that the child is a file but not a directory, only one Cloud Storage object will need to be stated. Similarly if the child is a directory but not a file.
 
 The behavior of type cache is controlled by the following flags/config parameters:
-1.  Type-cache TTL: It controls the duration for which Cloud Storage FUSE allows the kernel to inode type attributes. It can be set in one of the following two ways.
+1. Type-cache size: This is configurable at mount-level level by setting `metadata-cache: type-cache-max-size-mb` in config-file. This is the maximum size of type-cache in MiBs across a GCSFuse mount. By default, this is set at 32, which roughly equates to about a million entries. If you have more than a million objects (including files, directories, symlinks) across your bucket/buckets (one bucket in case of static mount and, multiple buckets in case of dynamic-mount), that you are going to access through this mount, then you may want to set this higher, otherwise the caching will not function properly when listing the contents:
```

Comment:
let's only write 
1. Type-cache size: This is configurable at mount-level level by setting `metadata-cache: type-cache-max-size-mb` in config-file. This is the maximum size of type-cache in MiBs across a GCSFuse mount. By default, this is set at 32, which roughly equates to about a million entries.

The other information should go in user recommendation doc about configuring type and stat cache.

Reply:
Also add average size for an entry in the doc in brackets. 
Also explain here how you found average size of an entry

Reply:
Done.